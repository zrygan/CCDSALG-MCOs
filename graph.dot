graph {
 Bruce -- Diana;
 Diana -- Clark;
 Diana -- Hal;
 Clark -- Hal;
}