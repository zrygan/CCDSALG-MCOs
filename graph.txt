6
A B C D -1
B A C E -1
C A B D E -1
D A C F -1
E B C F -1
F D E -1