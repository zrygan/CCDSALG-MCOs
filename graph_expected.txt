        Bruce   Clark   Diana   Hal
Bruce   0       0       1       0
Clark   0       0       1       1
Diana   1       1       0       1
Hal     0       1       1       0